### ISSUE TITLE
Javadocs geneation with Gradle

### ISSUE THREAD

**Issue by tiwanari** [PileProject/drive#56]
### Problem
`Drive` has some Javadocs but we don't have a good way to generate them.

### Proposal
Do Javadocs generation as a task of Gradle. It should be something like this;

```
./gradlew generateNxtJavadocs
```

**Comment by tiwanari** [comment]
I'm trying to use [vanniktech/gradle-android-javadoc-plugin: Gradle plugin that generates Java Documentation from an Android Gradle project.](https://goo.gl/BozlQu) as @myusak suggested offline but I got some errors related to auto-generated classes such as android's `R` and `com.pileproject.drive.preferences.BlockPreferences` ...  :cry:
(I also get some minor errors)

**Comment by tiwanari** [comment]
This is all I changed;

``` diff
diff --git app/build.gradle app/build.gradle
index 27c36b4..9b67cf6 100644
--- app/build.gradle
+++ app/build.gradle
@@ -16,6 +16,7 @@

 apply plugin: 'com.android.application'
 apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'com.vanniktech.android.javadoc'

 android {
     compileSdkVersion 23
diff --git build.gradle build.gradle
index 53c9d7d..542ffdc 100644
--- build.gradle
+++ build.gradle
@@ -3,11 +3,15 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.1.0'

         classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+
+        classpath 'com.vanniktech:gradle-android-javadoc-plugin:0.2.1'
+
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
```

**Comment by tiwanari** [comment]
I fixed these errors on #62.
